recipes/conversation_view/feed_item_row/feed_item_row.stories.js

```diff
@@ -0,0 +1,141 @@
+
+  <!-- @slot example slot decorator -->
+*/
+export const argTypesData = {
+  // Slots
+  /*
+    We use the following naming scheme `<SLOT_NAME>Slot` for slot controls to prevent conflicts with props that share
+    the same name. We provide the correct name of the slot using the name control attribute to ensure that the argument
+    table and description within the controls accurately reflects the correct names of our component's props and slots.
+  */
+  default: {
+    control: 'text',
+    table: {
+      type: {
+        summary: 'VNode',
+      },
+    },
+  },
+
+  threading: {
+    name: 'threading',
+    control: 'text',
+    table: {
+      category: 'slots',
+      type: {
+        summary: 'VNode',
+      },
+    },
+  },
+
+  reactions: {
+    name: 'reactions',
+    control: 'text',
+    table: {
+      category: 'slots',
+      type: {
+        summary: 'VNode',
+      },
+    },
+  },
+
+  menu: {
+    name: 'menu',
+    options: iconsList,
+    table: {
+      category: 'slots',
+      type: { summary: 'component' },
+    },
+    control: {
+      type: 'select',
+      labels: {
+        undefined: '(empty)',
+      },
+    },
+  },
+};
+
+// Story Collection
+export default {
+  title: 'Recipes/Conversation View/Feed Item Row',
+  component: DtRecipeFeedItemRow,
+  args: argsData,
+  argTypes: argTypesData,
+  excludeStories: /.*Data$/,
+  parameters: {
+    docs: {
+      page: DtRecipeFeedItemRowMdx,
+    },
+    controls: {
+      sort: 'requiredFirst',
+    },
+    options: { showPanel: true },
+  },
+};
+
+// Templates
+const DefaultTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtRecipeFeedItemRowDefaultTemplate,
+);
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtRecipeFeedItemRowVariantsTemplate,
+);
+
+// Stories
+export const Default = DefaultTemplate.bind({});
+Default.args = {
+  showHeader: true,
+  avatarImageUrl: 'https://i1.sndcdn.com/avatars-000181324408-652e57-t500x500.jpg',
+  displayName: 'Dwight Schrute',
+  time: '4:54 PM',
+  shortTime: '4:54',
+  onFocus: action('onFocus'),
+  onHover: action('onHover'),
+  default: `<p>Elementum fames nullam elementum velit proin vitae aliquet.
+  Platea nulla consectetur consequat sagittis nullam et ultricies nisl rhoncus
+  aliquet elementum venenatis quisque.</p>`,
+  menu: 'menu',
+  reactions: 'emoji reactions',
+  threading: 'threading',
+};
+
+export const Variants = VariantsTemplate.bind({});
+Variants.args = {
+  avatarImageUrl: 'https://i1.sndcdn.com/avatars-000181324408-652e57-t500x500.jpg',
+  displayName: 'Dwight Schrute',
+  time: '4:54 PM',
+  shortTime: '4:54',
+  onFocus: action('onFocus'),
+  onHover: action('onHover'),
+  default: `<p>Elementum fames nullam elementum velit proin vitae aliquet.
+  Platea nulla consectetur consequat sagittis nullam et ultricies nisl rhoncus
+  aliquet elementum venenatis quisque.</p>`,
+  menu: 'menu',
+};
```

Comment:
the args that are common to both Default and Variants you can put at the top in the "argsData" object. The args you put here should only be the ones that differ per story.